src/controllers/applicant.ts

```diff
@@ -14,14 +18,32 @@ export const createApplicant = async (req, res) => {
     });
 
     try {
-        await new_applicant.save();
+        await new_applicant.save(async () => {
+            const new_grade: ISceeningGrade = new ScreeningGrade({
```

Comment:
TS naming convention for our repo is camelCase, so these would be newApplicant and newGrade